python/lbann/util/__init__.py

```diff
@@ -28,3 +29,23 @@ def make_nd_array(*dims):
         return [make_nd_array(*tail) for _ in range(head)]
     else:
         return None
+
+def nvprof_command(nvprof_exe="nvprof",
+                   work_dir=None,
+                   output_name=None):
+    """Return the nvprof command and its arguments as a list.
+
+    Args:
+        nvprof_exe (str, optional): nvprof executable.
+        work_dir (str, optional): Working directory.
+        output_name (str, optional): File name to output nvprof trace.
+
+    """
+
+    if output_name is None:
+        output_name = "prof_%q{OMPI_COMM_WORLD_RANK}.nvvp"
```

Comment:
This is somewhat less clear, but supports non-OpenMPI systems:
```suggestion
        output_name = "prof-%h-%p.nvvp"
```